technical-reports/resolver/additional-context-and-information.md

```diff
@@ -1,78 +0,0 @@
-# Additional Context and Information
-
-## Resolution Aliasing
-
-Aliasing allows for dynamic namespacing or renaming of token paths during resolution. This is particularly useful when integrating external token sets or avoiding naming conflicts.
-
-</p>
-
-<aside class="example">
-
-Given a token set size.json:
-
-```json
-{
-  "sm": {
-    "value": "1px",
-    "type": "dimension"
-  },
-  "lg": {
-    "value": "10px",
-    "type": "dimension"
-  }
-}
-```
-
-By applying an alias in the modifier's meta.alias, we can namespace these tokens:
-
-```json
-{
-  "modifiers": [
-    {
-      "name": "size",
-      "type": "include",
-      "values": [
-        {
-          "name": "default",
-          "values": ["size.json"]
-        }
-      ],
-      "meta": {
-        "alias": "spacing"
-      }
-    }
-  ]
-}
-```
-
-Resulting in tokens accessible via spacing.sm and spacing.lg.
-
-</aside>
-
-<aside class="example">
-
-### Real-World Use Case: GitHub Primer
```

Comment:
For now, I removed this block since this is merely a reference to a design system, without clear relevance to the specification. It didn’t really come up in other examples.